src/archivist/fetcher/index.js

```diff
@@ -4,10 +4,23 @@ import fetchHtmlOnly from './htmlOnlyFetcher.js';
 export { launchHeadlessBrowser, stopHeadlessBrowser } from './fullDomFetcher.js';
 export { FetchDocumentError } from './errors.js';
 
-export default async function fetch({ url, executeClientScripts, cssSelectors }) {
+/**
+ * Fetch a resource from the network, returning a promise which is fulfilled once the response is available
+ *
+ * @param {Object} config - Fetcher configuration
+ * @param {string} config.url - URL of the resource you want to fetch
```

Comment:
Native objects should also be capitalised 🙂 

```suggestion
 * @param {String} config.url - URL of the resource to be fetched
```

Reply:
It seems that `boolean`, `string`, `number` are not capitalised https://jsdoc.app/tags-param.html

Reply:
Thanks, that'll teach me doing from memory instead of taking the time to double-check… I didn't hesitate enough, sorry for that!